src/components/Common/Breadcrumbs.tsx

```diff
@@ -109,7 +109,7 @@ export default function Breadcrumbs({
           <Button
             asChild
             variant="link"
-            className="p-1 font-normal text-gray-800 underline underline-offset-2 hover:text-gray-700"
+            className="p-0 font-normal text-gray-800 underline underline-offset-2 hover:text-gray-700"
```

Comment:
Why not remove the class itself 🤔 or does Button adds padding? 🤔

Reply:
Button adds padding